includes/wizards/class-engagement-wizard.php

```diff
@@ -126,6 +126,34 @@ public function register_api_endpoints() {
 				'permission_callback' => [ $this, 'api_permissions_check' ],
 			]
 		);
+		register_rest_route(
+			NEWSPACK_API_NAMESPACE,
+			'/wizard/' . $this->slug . '/social/meta_pixel',
+			[
+				[
+					'methods'             => \WP_REST_Server::READABLE,
+					'callback'            => [ 'Newspack\Meta_Pixel', 'api_get' ],
+					'permission_callback' => [ $this, 'api_permissions_check' ],
+				],
+				[
+					'methods'             => \WP_REST_Server::EDITABLE,
+					'callback'            => [ 'Newspack\Meta_Pixel', 'api_save' ],
+					'permission_callback' => [ $this, 'api_permissions_check' ],
+					'args'                => [
+						'active'   => [
+							'type'              => 'boolean',
+							'required'          => true,
+							'validate_callback' => [ 'Newspack\Meta_Pixel', 'validate_active' ],
+						],
+						'pixel_id' => [
+							'type'              => [ 'integer', 'string' ],
+							'required'          => true,
+							'validate_callback' => [ 'Newspack\Meta_Pixel', 'validate_pixel_id' ],
```

Comment:
When the option is not yet saved, an API request will be sent on clicking the toggle. This request will fail, since the `pixel_id` is invalid (it's `""` by default). This makes the toggle un-toggleable. 

<img width="790" alt="image" src="https://user-images.githubusercontent.com/7383192/191298556-c2141c54-33c4-44a1-bf2f-f50003709a8a.png">